test-infra-odoo/pom.xml

```diff
@@ -13,7 +13,7 @@
   <parent>
     <groupId>com.ozonehis</groupId>
     <artifactId>eip-odoo-openmrs-parent</artifactId>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.1.0-SNAPSHOT</version>
```

Comment:
```suggestion
    <version>${project.version}</version>
```

Reply:
Inside parent definition properties can't be added @corneliouzbett 
`Properties in parent definition are prohibited `